### DISCUSSION TITLE
What is "Connect two objects" for? (Left Panel, last item)

### DISCUSSION THREAD

**Discussion by FDK7** [SVG-Edit/svgedit#954] [Q&A] [answered]
What is "Connect two objects" for? (Left Panel, last item)

**Comment by DJ-asm** [comment, marked as answer]
Eye dropper tool is the last one, connect two objects is next to last. It makes a flow chart line between two shapes. To use, click on "Connect two objects", then one object (short line appears inside object)m then other object (lines moves to final position).

**Reply by hcooper** [reply]
![image](https://github.com/user-attachments/assets/26362856-7971-4cbe-85a0-4d91c4e04d1e)
it seems to depend on browser. In Firefox is appears last, and also doesn't work :-(
